src/cljc/org/candelbio/multitool/core.cljc

```diff
@@ -447,17 +459,25 @@
   [seq]
   (concat seq (repeat nil)))
 
+;;; I thought split-with did this, but no. Surely this in core?
+(defn divide-with
+  [p coll]
+  (let [groups (group-by (comp true? p) coll)]
+    [(get groups true) (get groups false)]))
+
 (defn mapf
```

Comment:
Like keep from core, but this uses nullish? where keep just checks for clojure falseyness